src/components/AppRoute.jsx

```diff
@@ -29,13 +29,19 @@ const AppRoute = (
     <Route component={EnsureHasAccounts}>
       <Route component={App}>
         <Redirect from="/" to={defaultRoute()} />
-        <Route path="balances" component={Balance} />
-        <Route path="transactions" component={TransactionsPage} />
+        <Route path="balances">
+          <IndexRoute component={Balance} />
+          <Route
+            path="reimbursements"
+            component={TransactionsPageWithBackButton}
+          />
+        </Route>
+        <Route path="transactions" component={TransactionsPageWithBackButton} />
         <Route path="categories">
           <IndexRoute component={CategoriesPage} />
           <Route
             path=":categoryName/:subcategoryName"
-            component={TransactionsPage}
+            component={props => <TransactionsPage {...props} showBackButton />}
```

Comment:
I think it's `TransactionsPageWithBackButton` to.

Reply:
Yes, thanks.